pkg/storage/couchdb/couchdbstore.go

```diff
@@ -227,3 +231,79 @@ func (c *CouchDBStore) getDataFromAttachment(k string) ([]byte, error) {
 
 	return data, nil
 }
+
+// Iterator returns an iterator that can be used to iterate over all the stored key-value pairs.
+func (c *CouchDBStore) Iterator() storage.StoreIterator {
```

Comment:
I think you need key range. get all docs is performance  problem

Reply:
@bstasyszyn  @troyronda  @sandrask

Reply:
I have no use for a key range, though. The EDV will need to check each and every stored document.

Reply:
Unless there's another, smarter way to query the database to find a specific value? @bstasyszyn @fqutishat

Just keep in mind this will need to work for memstore and couchdbstore

Reply:
@troyronda ^^^^^^